pom.xml

```diff
@@ -2142,7 +2153,7 @@ under the License.
                   <version>${targetJdk}</version>
                 </requireJavaVersion>
                 <requireMavenVersion>
-                  <version>3.5.0</version>
+                  <version>[3.5.0,)</version>
```

Comment:
I don't think this change is required: I am normally using Maven 3.6.0 without any issue.